### DISCUSSION TITLE
How to display hasMany childre in a list Table

### DISCUSSION THREAD

**Discussion by MWA-NASMO** [filamentphp/filament#6020] [Help]
Hello,
I'm new to Laravel 10 with FilamentPHP V2 and I'm stuck with displaying the children of a record in my Table.

I have a company that have multiple contacts and in the list of companies, I would like to show the contacts for that company Like :
**Companies**          |             **Contacts**
Name   1                     |          Contact 1                   Contact2
                                   |          address                     address
                                   |          email                         email
                                   |         Phone                        Phone
-----------------------------------------------------------------
Name   2                     |          Contact 1                   Contact2
                                   |          address                     address
                                   |          email                         email
                                   |         Phone                        Phone
-----------------------------------------------------------------
Is there anyone to help me ? I don't now if I have tu use widgets or anything else...

**Comment by danharrin** [comment]
Probably using this https://filamentphp.com/docs/2.x/tables/layout#custom-html
